rosa_hcp/rosa-hcp-deleting-cluster.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-hcp-deleting-cluster"]
+= Deleting a {hcp-title} cluster
+:context: rosa-hcp-deleting-cluster
+
+toc::[]
+
+This document provides steps to delete a {hcp-title-first} cluster that uses AWS Security Token Service (STS). After deleting your cluster, you can also delete the AWS Identity and Access Management (IAM) resources that are used by the cluster.
+
+include::modules/rosa-hcp-deleting-cluster.adoc[leveloffset=+1]
+
+.Troubleshooting
+* If the cluster cannot be deleted because of missing IAM roles, see xref:../support/troubleshooting/rosa-troubleshooting-deployments.adoc#rosa-troubleshooting-cluster-deletion_rosa-troubleshooting-cluster-deployments[Repairing a cluster that cannot be deleted].
+* If the cluster cannot be deleted for other reasons:
+** Ensure that there are no add-ons for your cluster pending in the link:https://console.redhat.com/openshift[Hybrid Cloud Console].
+** Ensure that all AWS resources and dependencies have been deleted in the Amazon Web Console.
+
+include::modules/rosa-deleting-sts-iam-resources-account-wide.adoc[leveloffset=+1]
+include::modules/rosa-deleting-account-wide-iam-roles-and-policies.adoc[leveloffset=+2]
+include::modules/rosa-unlinking-and-deleting-ocm-and-user-iam-roles.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+[id="additional-resources_rosa-hcp-deleting-cluster"]
+== Additional resources
+
+* For information about the AWS IAM resources for ROSA clusters that use STS, see xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-about-iam-resources[About IAM resources for ROSA clusters that use STS].
+* For information on cluster errors that are due to missing IAM roles, see xref:../support/troubleshooting/rosa-troubleshooting-deployments.adoc#rosa-troubleshooting-cluster-deletion_rosa-troubleshooting-cluster-deployments[Repairing a cluster that cannot be deleted].
```

Comment:
If you follow my suggestion above to use the trailing include for repairing clusters, I think that the following is sufficient, since the title of the IAM resources doc does a good job of explaining what it is about:
```suggestion
* xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-about-iam-resources[About IAM resources for ROSA clusters that use STS].
```

The definitive guidance on how to format Additional resources links is in this part of the doc guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

The newer resource linked above is handy for the other other examples it explains. But the guidance needs to be fixed so it doesn't contradict this older part of the guidelines.